build_scripts/dependencies_static_analysis_test_cov.py

```diff
@@ -109,14 +113,27 @@ def test_coverage():
     print("Stage test & coverage -- COMPLETED & PASSED --")
 
 
+def mutation_pass_fail_check():
+    """ Gates mutation test for 20 percentage """
+    call_subprocess("mutmut junitxml --suspicious-policy=ignore --untested-policy=ignore > mutmut.xml")
+    call_subprocess("python3 build_scripts/mutmut_parse.py --m 20")
```

Comment:
Not sure if "20" should be hardcoded, or should come from a configuration file. Your thoughts?

Reply:
it is a cmmandline option to the script. As same workflow all gating values in the buildscripts are commandline option which gets called in this script.